### PR TITLE
[RHCLOUD-23551] update Ansible templates

### PR DESCRIPTION
**Pull request by g-duval**
image-2023-01-12-15-38-43-829

### CODE REVIEW

File: engine/src/main/resources/templates/Ansible/instantEmailBodyV2.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{#content-title-section1}
+    Report available
+{/content-title-section1}
+{#content-body-section1}
+    <p>Ansible report available for download.</p>
+{/content-body-section1}
+{#content-button-section1}
+    <a href="{environment.url}/ansible/insights/reports/{action.context.slug}">Download now</a>
+{/content-button-section1}
+{/include}
```
Comment (josejulio) [inline engine/src/main/resources/templates/Ansible/instantEmailBodyV2.html:16]: I :heart:  that this one is so simple

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java [hunk 2]
```diff
@@ -25,6 +25,8 @@
 @ApplicationScoped
 public class TemplateService {
 
+    public static final String HCC_LOGO_TARGET = "Logo-Red_Hat-Hybrid_Cloud_Console-A-Reverse-RGB";
```
Comment (josejulio) [inline engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java:28]: This one is only used in the tests, could we move the variable there? (test code class if it's used by multiple tests)

**Review by josejulio — CHANGES_REQUESTED** [review]

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java (hunk 2 above)
Comment (g-duval) [reply engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java]: You are right, I moved it into TestHelper class which is much more appropriate.

File: engine/src/main/resources/templates/Ansible/instantEmailBodyV2.html (hunk 1 above)
Comment (g-duval) [reply engine/src/main/resources/templates/Ansible/instantEmailBodyV2.html:16]: Thanks

**Review by josejulio — APPROVED** [review]
lgtm

File: engine/src/test/java/com/redhat/cloud/notifications/TestHelpers.java [hunk 3]
```diff
@@ -327,4 +327,25 @@ public static Action createComplianceAction() {
 
         return emailActionMessage;
     }
+
+    public static Action createAnsibleAction(String slug) {
```
Comment (gwenneg) [inline engine/src/test/java/com/redhat/cloud/notifications/TestHelpers.java:331]: image

**Pull request merged**